clang/lib/Sema/SemaTemplateInstantiate.cpp

```diff
@@ -1348,6 +1348,16 @@ std::optional<TemplateDeductionInfo *> Sema::isSFINAEContext() const {
   return std::nullopt;
 }
 
+static TemplateArgument
+getPackSubstitutedTemplateArgument(Sema &S, TemplateArgument Arg) {
```

Comment:
`getSubstitutedTemplateArgumentPattern`  would be a better name. Please add a comment too